dali/operators/reader/loader/loader.h

```diff
@@ -212,6 +220,17 @@ class Loader {
     }
     return SizeImpl();
   }
+  // Give the size of the data accessed externally from python
+  Index SizePadded() {
+    if (!loading_flag_) {
+      PrepareMetadata();
+    }
+    if (!pad_last_batch_) {
```

Comment:
nitpick: I think it'd read more natural if you rephrase the logi as
```
if (pad_last_batch) {
  // .. handle special case
} else {
  return SizeImpl();
}
```

Reply:
Done